pkg/plugin/v3/api.go

```diff
@@ -125,6 +132,9 @@ func (p *createAPIPlugin) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&p.resource.Group, "group", "", "resource Group")
 	fs.StringVar(&p.resource.Version, "version", "", "resource Version")
 	fs.BoolVar(&p.resource.Namespaced, "namespaced", true, "resource is namespaced")
+
+	fs.StringVar(&p.crdVersion, "crd-version", defaultCRDVersion,
+		"CustomResourceDefinition version: [v1, v1beta1]")
```

Comment:
```suggestion
		"CRD apiVersion which will be used to do the scaffold. Options: [v1, v1beta1]")
```
IMO we need clarifies that it is the apiVersion attribute of the manifest.

Reply:
I think it is solved. So, I am closing this one.